test/tools/schema_validator/validator.h

```diff
@@ -2,12 +2,14 @@
 
 #include <string>
 
-#include "common/json/json_loader.h"
+#include "envoy/api/api.h"
+
+#include "common/stats/isolated_store_impl.h"
```

Comment:
Is this include needed?

Reply:
It's included because the Validator class has a `Stats::IsolatedStoreImpl` member that's used to instantiate `Api::ApiPtr` below.